Code/JavaWrappers/RWMol.i

```diff
@@ -109,8 +110,26 @@ static RDKit::RWMOL_SPTR MolFromPDBFile(std::string fName,
   mol=RDKit::PDBFileToMol(fName,sanitize,removeHs,flavor);
   return RDKit::RWMOL_SPTR(mol);
 }
+static RDKit::RWMOL_SPTR MolFromSequence(std::string text,
+                                  bool sanitize=true,int flavor=0){
+  RDKit::RWMol *mol=0;
```

Comment:
No, the pointer will get passed to the ROMOL_SPTR is what I meant.  This prevents a double reference in non cxx11 I think ( unless it is elided )